main.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable  no-unused-vars */
```

Comment:
- [ ] Kindly don't manually disable the `ESlint`. Instead, you should fix the problems with the code by yourself.
The Linters are helping to find issues with the code so that we fix them to keep it clean.